app/models/wishlist.rb

```diff
@@ -0,0 +1,4 @@
+class Wishlist < ApplicationRecord
+  validates :name, length: { in: 3..30 }
+  validates :description, length: { in: 10..150 }
```

Comment:
`validates :name, presence: true, length: { maximum: 50 }`
`validates :description, length: { maximum: 500 }`
IMHO would be enough. But it's your choice. :)